blueprints/default/flow-password-change.yaml

```diff
@@ -11,6 +11,15 @@ entries:
     slug: default-password-change
   model: authentik_flows.flow
   id: flow
+- attrs:
+    check_have_i_been_pwned: true
```

Comment:
as much as I would like to enable this by default, we probably shouldn't enable this by default for airgapped instance

Reply:
Fair enough. I was going by NIST's

> verifiers SHALL compare the prospective secret against a blocklist that contains known commonly used, expected, or compromised passwords

but on second thought, I agree, defaulting to HIBP is a step too far.

Perhaps we can do something about that later. I'll remove for now and add it to the hardening docs.

Reply:
we can set `check_zxcvbn` to `true` which includes a blocklist

Reply:
That seems to work okay. Added.

I didn't add `zxcvbn` initially because I felt it was too much restriction (based on NIST). However, a score of 2 seems like it effectively adds the blocklist and some variations, but not much more.